### PR TITLE
CHI-627: Update to Node 14

### PR DESCRIPTION
**Pull request by stephenhand**
Primary reviewer: @GPaoloni 

## Description

* Update runtime for all functions to Node 14
* Update all existing workflows to build with Node 14

### Checklist
- [ X ] Corresponding issue has been opened
- [ N/A ] New tests added
- [ N/A ] Strings are localized

### Related Issues
CHI-627

### Verification steps

Release Aselo Dev serverless

### CODE REVIEW

File: .github/workflows/test_staging.yml [hunk 1]
```diff
@@ -89,9 +89,9 @@ jobs:
...
           TWILIO_ACCOUNT_SID: ${{ secrets.TEST_STG_ACCOUNT_SID }}
           TWILIO_AUTH_TOKEN: ${{ secrets.TEST_STG_AUTH_TOKEN }}
-        run: npm install twilio-cli -g && twilio plugins:install @twilio-labs/plugin-serverless && twilio serverless:deploy --runtime=node12 --service-name=serverless --environment=dev --force
+        run: npm install twilio-cli -g && twilio plugins:install @twilio-labs/plugin-serverless && twilio serverless:deploy --runtime=node14 --service-name=serverless --environment=dev --force
       - name: Run deploy command to production
         env:
           TWILIO_ACCOUNT_SID: ${{ secrets.TEST_STG_ACCOUNT_SID }}
           TWILIO_AUTH_TOKEN: ${{ secrets.TEST_STG_AUTH_TOKEN }}
-        run: twilio serverless:deploy --runtime=node12 --service-name=serverless --environment=production --force
+        run: twilio serverless:deploy --runtime=node14 --service-name=serverless --environment=production --force
```
Comment (GPaoloni) [inline .github/workflows/test_staging.yml:97]: Question: Do we really need this workflow? Is this environment used? And if it is, why is not part of the standardized workflows?

**Review by GPaoloni — APPROVED** [review]
Looks great, thanks! I've been using node 14 for a while now and the functions always worked as expected, so I don't expect anything breaking in the actual runtime.

File: .github/workflows/test_staging.yml (hunk 1 above)
Comment (stephenhand) [reply .github/workflows/test_staging.yml:97]: I don't know? I've not seen reference to it elsewhere?

**Comment by GPaoloni** [comment]
@stephenhand I'll merge this now as Aselo developments are failing for me. I think this PR fixes the issue.

**Comment by stephenhand** [comment]
> @stephenhand I'll merge this now as Aselo developments are failing for me. I think this PR fixes the issue.

Sorry @GPaoloni - I should have said something, I was thinking this would be merged before the broken deploy caused any disruption - thanks for merging it

**Pull request merged**